src/client/components/Subheader/index.js

```diff
@@ -1,18 +1,42 @@
 import BPOButton from "../BPOButton";
+import Dropdown from "react-bootstrap/Dropdown";
 import React from "react";
+import i18n from "../../i18n";
 import { useTranslation } from "react-i18next";
 
 function Subheader() {
   const { t } = useTranslation();
 
+  const changeLanguage = (lng) => {
+    i18n.changeLanguage(lng);
+  };
+
   return (
     <div className="bg-light">
       <div className="container p-4">
         <h1>{t("subheaderHeader")}</h1>
         <p>{t("subheaderSubheader")}</p>
         <p>{t("subheaderParagraph")}</p>
         <p>{t("subheaderParagraph2")}</p>
-        <BPOButton />
+        <div className="btn-toolbar" role="toolbar">
+          <div className="mr-4 mb-4">
+            <BPOButton />
+          </div>
+          <Dropdown>
+            <Dropdown.Toggle variant="outline-secondary" className="text-dark">
+              Select Language
```

Comment:
I'm adding this language here for now instead of the translations files for two reasons:
1) We always want to display this in English, regardless of the language the user has chosen, and 
2) We're deploying this in the middle of the day, and I don't want the caching bug to affect any current users.

Reply:
This seems fine, but maybe follow up on the issue about whether this is the desired behavior or not.